netbox/ipam/filtersets.py

```diff
@@ -882,8 +930,9 @@ def search(self, queryset, name, value):
         return queryset.filter(qs_filter)
 
     def filter_scope(self, queryset, name, value):
+        model_name = name.replace('_', '')
         return queryset.filter(
-            scope_type=ContentType.objects.get(model=name),
+            scope_type=ContentType.objects.get(model=model_name),
```

Comment:
I assume this is a separate bug-fix?

Reply:
This change was just to accommodate renaming the two filters above (`sitegroup` and `clustergroup`).